module/zstd/README.md

```diff
@@ -29,30 +29,18 @@ To update ZSTD the following steps need to be taken:
    - `zstd/lib/zstd.h`
    - `zstd/lib/common/zstd_errors.h`
 
-This can be done using a few shell commands from inside the zfs repo:
-
-~~~sh
-cd PATH/TO/ZFS
-
-url="https://github.com/facebook/zstd"
-release="$(curl -s "${url}"/releases/latest | grep -oP '(?<=v)[\d\.]+')"
-zstd="/tmp/zstd-${release}/"
-
-wget -O /tmp/zstd.tar.gz \
-    "${url}/releases/download/v${release}/zstd-${release}.tar.gz"
-tar -C /tmp -xzf /tmp/zstd.tar.gz
-
-cp ${zstd}/lib/zstd.h module/zstd/lib/
-cp ${zstd}/lib/zstd_errors.h module/zstd/lib/
-${zstd}/contrib/single_file_libs/combine.sh \
-    -r ${zstd}/lib -o module/zstd/lib/zstd.c module/zstd/zstd-in.c
-~~~
+This can be done using a few shell commands from inside the zfs repo.
+"update_zstd.sh" is included as a scripted example of how to update the ZSTD library.
 
 Note: if the zstd library for zfs is updated to a newer version,
 the macro list in include/zstd_compat_wrapper.h usually needs to be updated.
-this can be done with some hand crafting of the output of the following
-script: nm zstd.o | awk '{print "#define "$3 " zfs_" $3}' > macrotable
+One can use the following script to create a list of required macros: 
+```
+nm zstd.o | awk '{print "#define "$3 " zfs_" $3}' > macrotable
+```
 
+This list then needs to be manually filtered for relevance, validity, duplicates, blank lines etc.
+_TODO: Add a script to automate some repeatitive bits of this a little more_
```

Comment:
`nm -fposix zstd.o | awk '{print "#define "$1 " zfs_" $1}' |  sort -u > macrotable`

^ this saves you from "empty lines" (I guess you mean such: "#define  zfs_") and drops duplicates

Reply:
Awesome, thanks... Didn't have the time yet to figure that out, will look into it/include it :)
(either in the .sh or the readme whatever the final choice would be)

Reply:
Changed, thanks!